docs/source/faq.rst

```diff
@@ -55,7 +55,7 @@ FAQ
 
 * What is a rollup?
 
-  A rollup performs transaction executioon oustide Layer 1 nad then the data is
+  A rollup performs transaction executioon oustide Layer 1 and then the data is
```

Comment:
My goodness... there are additional typos in here...

Reply:
wow, and I missed those...